frontend/src/DiscoverMovie/Overview/AddListMovieOverview.js

```diff
@@ -138,6 +143,26 @@ class AddListMovieOverview extends Component {
                 {...linkProps}
               >
                 {title}({year})
+
+                <span style={{ marginLeft: `${linkMenuMargin}em` }} className={styles.links}>
+                  <Popover
+                    anchor={
+                      <Icon
+                        name={icons.EXTERNAL_LINK}
+                        size={15}
+                      />
+                    }
+                    title="Links"
+                    body={
+                      <MovieDetailsLinks
+                        tmdbId={tmdbId}
+                        imdbId={imdbId}
+                        youTubeTrailerId={youTubeTrailerId}
+                      />
+                    }
+                  />
+                </span>
```

Comment:
Should we move this to be beside the exclude button on overviews
![image](https://user-images.githubusercontent.com/376117/87845414-0652e680-c895-11ea-8ad9-91f1fc082f92.png)